Realm/Realm/Schema/Property.cs

```diff
@@ -180,6 +181,7 @@ internal Property(in SchemaProperty nativeProperty)
         internal SchemaProperty ToNative(Arena arena) => new()
         {
             name = StringValue.AllocateFrom(Name, arena),
+            managed_name = StringValue.AllocateFrom(ManagedName != Name ? ManagedName : string.Empty, arena),
```

Comment:
You can initialize this with `null` instead of `string.Empty` - `capi_to_std(realm_string_t)` knows to return an empty `std::string` if the marshaled string buffer is null. This shaves off an empty, unnecessary allocation.